com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/AbstractProtoBuilderTest.java

```diff
@@ -175,6 +176,16 @@ protected List<Message> build(String file, String source) throws Exception {
         return messages;
     }
 
+    protected void clean() throws Exception {
+        Collection<String> result = new ArrayList<>();
+        fileSystem.walk("build", new FileSystemWalker() {
+            public void handleFile(String path, Collection<String> results) {
+                fileSystem.addFile(path, null);
+            }
+        }, result);
+        project.build(new NullProgress(), "clean");
+    }
```

Comment:
It's important to clean up the result of a build if it's necessary to build the same file twice. For Mock FS, we can't delete files; we just replace their content with `nil`.